packages/qwik/src/core/util/unitless_number.ts

```diff
@@ -0,0 +1,51 @@
+/** CSS properties which accept numbers but are not in units of "px". */
```

Comment:
Thanks for your help. Where did you find this list? is there an official site with this list?

Reply:
Hey @gioboa, unfortunately I couldn't find any official site that mentioned this but this list is referred from React's official codebase here: https://github.com/facebook/react/blob/main/packages/react-dom-bindings/src/shared/isUnitlessNumber.js